### ISSUE TITLE
[Bug]: Type error when build the storefront using yarn build

### ISSUE THREAD

**Issue by ymaheshwari1** [vuestorefront-community/medusa#3]
### Expected Behavior

The storefront should be built without any error

### Actual Behavior

Getting type error:
```
packages/composables/src/composables/useCart/index.ts(28,3): semantic error TS2322: Type '(context: Context<any, any, any>, { currentCart, product, quantity, customQuery }: { currentCart: Cart; product: Product; quantity: any; customQuery?: CustomQuery; }) => Promise<{}>' is not assignable to type '(context: Context<any, any, any>, params: { currentCart: Cart; product: Product; quantity: any; customQuery?: CustomQuery; }) => Promise<Cart>'.
  Type 'Promise<{}>' is not assignable to type 'Promise<Cart>'.
    Type '{}' is missing the following properties from type 'Cart': id, object, email, billing_address_id, and 26 more.
```

### Steps to reproduce

- Run yarn build command

### What version of Medusa.js integration are you using?

0.0.1

### What version of Node.js are you using?

14.0.0

### What browser (and version) are you using?

Chrome

### What operating system (and version) are you using?

windows

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hudejo** [comment]
Hi Yash,
Any improvement on this bug fixing?
I've received the same error on running 'yarn build:composables' command. My medusa version is 1.13.0
Thx, ;-D

**Comment by hudejo** [comment]
a quick fix for medusa/packages/composables/src/composables/useCart/index.ts:

    const cart = {} as Cart; 
    return cart;

instead of
    return {};

where it is required

**Comment by ymaheshwari1** [comment]
No, I didn't explored on the issue much, but the quick fix that you shared works.
Are you working on VSF Medusa integration? or you are just exploring around?

**Comment by hudejo** [comment]
Basically I would like to try it then yes, maybe integrating it... But could not get to that 1st level what E.g. nextjs template for medusajs provides: to have a minimal, but working front-end as well on top of medusajs. Can you drive me to the docs where I can get requirements for missing functionalities like addItem of useCart of Vue storefornt?
